master/buildbot/newsfragments/pushjet.feature

```diff
@@ -0,0 +1 @@
+Added a https://pushjet.io/ reporter as :py:class `buildbot.reporters.pushjet.PushjetNotifier`
```

Comment:
please use :bb:reporter:`PushjetNotifier` (also make sure there is no white space between : and `)